certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2953 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PPM Express",
+    "version": "v1.0",
+    "description": "PPM Express is Online Project Portfolio Management Software that was designed to easily integrate and orchestrate your existing projects in Microsoft Planner, Azure DevOps, Jira Software and Microsoft Project Online.",
+    "contact": {
+      "name": "PPM Express Support",
+      "url": "https://ppm.express/contact-us/",
+      "email": "support@ppm.express"
+    }
+  },
+  "host": "api-us.ppm.express",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-capabilities": {
+    "testConnection": {
+      "operationId": "Me_Get"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://ppm.express"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://ppm.express/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Business Management;Business Intelligence"
+    }
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/Challenges/{id}": {
+      "get": {
+        "tags": [
+          "Challenges"
+        ],
+        "summary": "Get a Challenge",
+        "description": "Gets an existing Challenge by Id",
+        "operationId": "Challenges_Get",

```

Comment:
@vsorokin-ppmexpress please use [pascal casing for operationId](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers) properties.